examples/user_guide/Geometry_Data.ipynb

```diff
@@ -0,0 +1,273 @@
+    "\n",
+    "In summary ``Polygons`` can be represented in much the same way as ``Paths`` above but have a special reserved key to store the polygon interiors or 'holes'. The holes are stored as a list-of-lists of arrays. This nested format is necessary to unambiguously associate holes with the sub-geometries in a multi-geometry. In the simplest case of a single Polygon geometry the format looks like this:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "xs = [1, 2, 3]\n",
+    "ys = [2, 0, 7]\n",
+    "holes = [[[(1.5, 2), (2, 3), (1.6, 1.6)], [(2.1, 4.5), (2.5, 5), (2.3, 3.5)]]]\n",
+    "\n",
+    "hv.Polygons([{'x': xs, 'y': ys, 'holes': holes}]).options(padding=0.1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The 'x' and 'y' coordinates represent the exterior of the Polygon and the list-of-list of holes defines two interior regions inside the polygon.\n",
+    "\n",
+    "In a multi-Polygon arrangement where two Polygon geometries are separated by NaNs, the purpose of the nested format becomes a bit clearer. Here the polygon from above still has the two holes but the second polygon does not have any holes, which we declare with an empty list:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "xs = [1, 2, 3, np.nan, 6, 7, 3]\n",
+    "ys = [2, 0, 7, np.nan, 7, 5, 2]\n",
+    "\n",
+    "holes = [\n",
+    "    [[(1.5, 2), (2, 3), (1.6, 1.6)], [(2.1, 4.5), (2.5, 5), (2.3, 3.5)]],\n",
+    "    []\n",
+    "]\n",
+    "\n",
+    "hv.Polygons([{'x': xs, 'y': ys, 'holes': holes}]).options(padding=0.1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If a polygon has no holes at all the 'holes' key may be ommitted entirely:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "hv.Polygons([{'x': xs, 'y': ys, 'holes': holes, 'value': 0},\n",
+    "             {'x': [4, 6, 6], 'y': [0, 2, 1], 'value': 1}], vdims='value').options(padding=0.1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Accessing the data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To access the underlying data the geometry elements (``Path``/``Contours``/``Polygons``) implement a ``split`` method. By default it simply returns a list of elements, where each contains only one geometry:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "poly = hv.Polygons([\n",
+    "    {'x': xs, 'y': ys, 'holes': holes, 'value': 0},\n",
+    "    {'x': [4, 6, 6], 'y': [0, 2, 1], 'value': 1}\n",
+    "], vdims='value')\n",
+    "\n",
+    "polys = poly.split()\n",
```

Comment:
Don't think it is worth introducing the `polys` handle here (just inline it to the `Layout`). Looks to me like the handle that is reused is just `poly`.

Reply:
Sounds good.